google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -20,8 +20,8 @@
   <!-- "version" set by gradle-intellij-plugin -->
   <!-- "idea-version since-build" set by gradle-intellij-plugin -->
   <!-- Workaround for gradle-intellij-plugin limitation: https://github.com/JetBrains/gradle-intellij-plugin/issues/66 -->
-  <!-- Setting version to 15.0.3 and up. https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+15+143.1821.5+Release+Notes -->
-  <idea-version since-build="143.1821.5" />
+  <!-- Setting version to 15.0.6 and up. https://confluence.jetbrains.com/display/IDEADEV/IntelliJ+IDEA+15+143.1821.5+Release+Notes -->
```

Comment:
Yea. I'm trying to find a good URL to link to. There aren't really good mappings between the full version numbers and the public ones (I determine them mainly by launching the desired version of IJ, e.g. 15.0.6 and then looking at the about page to get the full version number).